definitions.md

```diff
@@ -175,6 +191,12 @@ Used in the Filecoin system by a storage miner to prove that client data was kep
 
 TODO add a note to distinguish predictability from randomness
 
+#### Randomness Lookback
```

Comment:
I suggest a different name which is more specific, e.g. Leader Randomness Lookback, or Election Randomness Lookback.